### PR TITLE
chore: add r/demo/print

### PR DESCRIPTION
**Pull request by grepsuzette**
add /r/demo/print

This realm simply prints user address and similar information.          
It can help designing wallet connection.                                
                                                                        
See also r/demo/echo (to print various thing like user address)

### CODE REVIEW

**Comment by leohhhn** [comment]
@grepsuzette 

We have a whole saga on why `_test.pkg.bft` is failing. Ignore it :)

File: examples/gno.land/r/demo/print/gno.mod [hunk 1]
```diff
@@ -0,0 +1 @@
+module gno.land/r/demo/priint
```
Comment (thehowl) [inline examples/gno.land/r/demo/print/gno.mod:1]:
```suggestion
module gno.land/r/demo/print
```

File: examples/gno.land/r/demo/print/print.gno [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import (
+	"std"
+)
+
+/*
+ * This realm simply prints user address and similar information.
+ * It can help designing wallet connection.
+ *
+ * See also r/demo/echo (to print various thing like user address)
+ */
```
Comment (zivkovicmilos) [inline examples/gno.land/r/demo/print/print.gno:12]: Nitpick: please use `//` notation instead of `/**/`

**Review by zivkovicmilos — CHANGES_REQUESTED** [review]
I'm not sure I completely understand the usage of a Realm like this. What value does this add to the `examples` directory? I am also not completely sold on naming it `/r/demo/print`

**Pull request closed without merging**